HUnit.cabal

```diff
@@ -6,18 +6,17 @@ License-File:           LICENSE
 Author:                 Dean Herington
 Maintainer:             Simon Hengel <sol@typeful.net>
 Stability:              stable
-Homepage:               http://hunit.sourceforge.net/
+Homepage:               https://github.com/hspec/HUnit#hunit
```

Comment:
It may be more future proof to use https://github.com/hspec/HUnit#readme as the link to the readme, as it will continue to work even if we change the heading.